travis/update-release.py

```diff
@@ -121,8 +135,111 @@ def get_container_images_data(r_stream, tag):
             "severity": count_severity("release_artifacts/" + RELEASE_TAG + DIR + image["name"] + "/" + RELEASE_TAG + "-" + "cve.txt")
         }
         c_imagges.append(image_update)
+
     return c_imagges
 
+def check_rollback_artifacts(r_stream, search_stream,tag):
+    z_container_images = {}
+    for r in r_stream:
+        if r["release_name"].endswith(search_stream+".z"):
```

Comment:
keeping the above comment for get_container_images_data in mind, since we are only rolling back artifacts for a release (IS_RELEASE == true), the check works here:
if r["release_name"].endswith(search_stream+".z"), since your search_stream is 4.3.2.1 and you are looking for 4.3.2.1.z this works